corgi/core/migrations/0039_auto_20221013_0748.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 3.2.15 on 2022-10-13 07:48
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("core", "0038_add_olcs_fields"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="softwarebuild",
+            name="completion_time",
+            field=models.DateTimeField(null=True),
+        ),
+        migrations.AlterField(
+            model_name="component",
+            name="software_build",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name="components",
+                to="core.softwarebuild",
+            ),
+        ),
+        migrations.AddIndex(
+            model_name="softwarebuild",
+            index=models.Index(fields=["completion_time"], name="core_softwa_complet_c38fc7_idx"),
+        ),
+    ]
```

Comment:
It would be nice to add a data migration here to populate all the completion dates for all builds instead of relying on save().

Reply:
sure will add it